Index: Passman/obj/rider.project.model.nuget.info
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>16958105284291906
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Passman/obj/rider.project.model.nuget.info b/Passman/obj/rider.project.model.nuget.info
--- a/Passman/obj/rider.project.model.nuget.info	
+++ b/Passman/obj/rider.project.model.nuget.info	
@@ -1,1 +1,1 @@
-16958105284291906
\ No newline at end of file
+16959028830998119
\ No newline at end of file
